src/main/java/de/zalando/aruha/nakadi/service/subscription/model/Session.java

```diff
@@ -0,0 +1,42 @@
+package de.zalando.aruha.nakadi.service.subscription.model;
+
+import java.util.UUID;
+
+public class Session {
+    public final String id;
```

Comment:
I don't think that in this case encapsulation is really would be encapsulation. It will be just a set of getters, that will add additional calls.
I'll fix it in another way, so it will be simpler to use and will save us time instead of staring holly war.